project/ValidatePullRequest.scala

```diff
@@ -209,4 +208,13 @@ object ValidatePullRequest extends AutoPlugin {
     // add reportBinaryIssues to validatePullRequest on minor version maintenance branch
     validatePullRequest <<= validatePullRequest.dependsOn(reportBinaryIssues)
   )
+
+  def whenValidatingPr(task: TaskKey[_]) =
+    validatePullRequest := {
+      (projectBuildMode in ValidatePR).value match {
+        case BuildSkip => // do not run multi-jvm tests if project is skipped during pr validation
+        case _ => task.value
+      }
+      validatePullRequest.value
+    }
```

Comment:
Nice one! Slightly adjust the comment maybe to explain "when BuildSkip we do not do anything here" and "task.value" will force the tasks execution